src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -895,7 +895,9 @@ protected override CompilationOptions CommonWithCheckOverflow(bool checkOverflow
         [EditorBrowsable(EditorBrowsableState.Never)]
         public CSharpCompilationOptions(
             OutputKind outputKind,
+#pragma warning disable IDE0060 // Remove unused parameter
             bool reportSuppressedDiagnostics,
+#pragma warning restore IDE0060 // Remove unused parameter
```

Comment:
I was surprised that explicit intervention was needed to avoid this warning on public methods. I thought it would see that it was public API And automatically know we can't make a change.